articles/openshift/howto-create-private-cluster.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Create a private cluster with Azure Red Hat OpenShift 3.11 | Microsoft Docs
+description: Create a private cluster with Azure Red Hat OpenShift 3.11
+author: klamenzo 
+ms.author: b-lejaku
+ms.service: container-service
+ms.topic: conceptual
+ms.date: 03/02/2020
+keywords: aro, openshift, privatecluster, red hat
+#Customer intent: As a customer, I want to create a private cluster on ARO OpenShift.
+---
+
+# Create a private cluster with Azure Red Hat OpenShift 3.11
+
+> [!IMPORTANT]
+> ARO private clusters are currently only available in private preview in East US 2. Private preview acceptance is by invitation only. Please be sure to register your subscription before attempting to enable this feature.
```

Comment:
Is this still true? I know we've got 4.3 private cluster docs for OpenShift here - https://docs.openshift.com/container-platform/4.3/installing/installing_azure/installing-azure-private.html

Reply:
Let me confirm and get back to you. I think that documentation is for private cluster for OpenShift on Azure as Iaas.

Reply:
Yup, still preview and avail only in eastus and europe